autoload/sandwich/magicchar/t.vim

```diff
@@ -129,6 +129,13 @@ function! s:tokenize_custom_attributes(text, j) abort "{{{
   return [j - a:j, tokenlist]
 endfunction
 "}}}
+
+
+function! s:matches_filetype(filetype, list) abort "{{{
+    return index(a:list, a:filetype) != -1
```

Comment:
Excuse me I missed last time but please unify the indent to two spaces here also.